Description: Cleanup for possible privacy breaches
Author: Markus Frosch <markus@lazyfrosch.de>
Last-Update: 2013-12-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/docs/de_DE/toc.html
+++ b/docs/de_DE/toc.html
@@ -77,7 +77,7 @@
 -->
         <div style="text-align:center;margin-top:20px;">
         <p><a href="http://www.nagvis.org" style="margin: 0px;" class="nostyle"><img width="88" height="23" border="0" alt="NagVis Logo" src="../general/img/logo-88x23.png" style="margin: 0px;" /></a></p>
-        <p><a href="http://sourceforge.net" style="margin: 0px;" class="nostyle"><img width="88" height="31" border="0" alt="SourceForge.net Logo" src="http://sflogo.sourceforge.net/sflogo.php?group_id=132019&amp;type=1" style="margin: 0px;" /></a></p>
+        <p><a href="http://sourceforge.net" style="margin: 0px;" class="nostyle">SourceForge.net</a></p>
         </div>
     </body>
 </html>
--- a/docs/en_US/toc.html
+++ b/docs/en_US/toc.html
@@ -68,7 +68,7 @@
         <p>(<font style="color:#ff0000;">*</font>) New in 1.7</p>
         <div style="text-align:center;margin-top:20px;">
         <p><a href="http://www.nagvis.org" style="margin: 0px;" class="nostyle"><img width="88" height="23" border="0" alt="NagVis Logo" src="../general/img/logo-88x23.png" style="margin: 0px;" /></a></p>
-        <p><a href="http://sourceforge.net" style="margin: 0px;" class="nostyle"><img width="88" height="31" border="0" alt="SourceForge.net Logo" src="http://sflogo.sourceforge.net/sflogo.php?group_id=132019&amp;type=1" style="margin: 0px;" /></a></p>
+        <p><a href="http://sourceforge.net" style="margin: 0px;" class="nostyle">SourceForge.net</a></p>
         </div>
     </body>
 </html>
